### PR TITLE
MAT-3905, service tests for MAT-3903

### PR DESCRIPTION
**Pull request by praveenavas**
MAT-3905, service tests for MAT-3903

### CODE REVIEW

File: cypress/Shared/CQLLibraryPage.ts [hunk 1]
```diff
@@ -4,18 +4,48 @@ export class CQLLibraryPage {
 
     public static readonly createCQLLibraryBtn = '[data-testid="create-new-cql-library-button"]'
     public static readonly cqlLibraryNameTextbox = '[data-testid="cql-library-name-text-field"]'
+    public static readonly cqlLibraryModelDropdown = '[class="MuiSelect-select MuiSelect-outlined MuiOutlinedInput-input MuiInputBase-input MuiInputBase-inputSizeSmall css-182didf"]'
```
Comment (cvasile001) [inline cypress/Shared/CQLLibraryPage.ts:7]: can you request some IDs for these elements?

**Review by cvasile001 — CHANGES_REQUESTED** [review]
Lets request the developer of the UI to add IDs to these elements

File: cypress/Shared/CQLLibraryPage.ts (hunk 1 above)
Comment (praveenavas) [reply cypress/Shared/CQLLibraryPage.ts:7]: ok

**Review by cvasile001 — APPROVED** [review]
All good

**Pull request merged**